atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueue.java

```diff
@@ -241,10 +260,27 @@ static SweepQueueFactory create(
                 Supplier<Integer> shardsConfig,
                 TransactionService transaction,
                 ReadBatchingRuntimeContext readBatchingRuntimeContext) {
+            return create(
```

Comment:
We do like to use telescopic constructors when adding a new parameter to avoid breaking things! 

However, would we be breaking things if we just added a new parameter here and pulled the default refresh time into the caller(s)? If there's multiple, it may be less readable, but are there multiple?

If you'd like help figuring out the answer to this, let me know. There are some internal tools you can use to make this much easier :)

Reply:
(Although again, you may not even need this if you pull out the task, since you then don't need to pass in a manual refreshMillis )